pm_icecon/nt/compute_nt_ic.py

```diff
@@ -177,34 +177,93 @@ def apply_nt_spillover(
     *, conc: npt.NDArray, shoremap: npt.NDArray, minic: npt.NDArray
 ) -> npt.NDArray[np.int16]:
     """Apply the NASA Team land spillover routine."""
+    """
+      conc is 0 to > 100.0
+        (per this code, negative value means missing data)
+
+      shoremap is 0: ocean, 1: land, 2: coast,
+                   3: coast, 4: near_coast, 5: far_coast
+        in original, shoremap is big-endian int16, but nothing in the
```

Comment:
We should clarify what "original" means here, or remove this note.

Reply:
This would be the original source code from Goddard.

Reply:
We should probably add some documentation about the history of this project, and where the original algorithms came from.

There is a long history here:

* Goddard originally developed the code
* NSIDC developed `pmalgos` in the late ~1990s from the goddard code
* More recently, we developed `cdralgos`, based on `pmalgos` and the newly updated goddard code (Goddard also continued to develop and run their own code)
* Now this!